physics/cu_gf_driver.F90

```diff
@@ -200,7 +205,7 @@ subroutine cu_gf_driver_run(ntracer,garea,im,km,dt,cactiv,                &
 !  qfx2 -- latent heat flux (kg/kg m/s), positive upward from sfc 
 !  gf needs them in w/m2. define hfx and qfx after simple unit conversion
    real(kind=kind_phys), dimension (im)  :: hfx,qfx
-   real(kind=kind_phys) tem,tem1,tf,tcr,tcrf
+   real(kind=kind_phys) trash,tem,tem1,tf,tcr,tcrf
```

Comment:
I don't see `trash` being used.

Reply:
I just updated the PR without that variable, thanks